Docker build: Failed to load '/grafana/packages/grafana-plugin-configs/scripts/webpack/webpack.prod.js' config

### What happened?

When building grafana in [Dockerfile](https://github.com/grafana/grafana/blob/81fd8f4126bfa15d5bf4e0defe62af176a0ff4cf/Dockerfile)

```
#51 [js-builder 14/14] RUN yarn build
#51 3.169 [webpack-cli] Failed to load '/tmp/grafana/packages/grafana-plugin-configs/scripts/webpack/webpack.prod.js' config
#51 3.174 [webpack-cli] Error: Cannot find module '/tmp/grafana/packages/grafana-plugin-configs/scripts/webpack/webpack.prod.js'
#51 3.174 Require stack:
#51 3.174 - /tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js
#51 3.174 - /tmp/grafana/node_modules/webpack-cli/lib/bootstrap.js
#51 3.174 - /tmp/grafana/node_modules/webpack-cli/bin/cli.js
#51 3.174 - /tmp/grafana/node_modules/webpack/bin/webpack.js
#51 3.174     at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
#51 3.174     at Module._load (node:internal/modules/cjs/loader:984:27)
#51 3.174     at Module.require (node:internal/modules/cjs/loader:1231:19)
#51 3.174     at require (node:internal/modules/helpers:179:18)
#51 3.174     at WebpackCLI.tryRequireThenImport (/tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js:223:30)
#51 3.174     at loadConfigByPath (/tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js:1406:38)
#51 3.174     at /tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js:1460:88
#51 3.174     at Array.map (<anonymous>)
#51 3.174     at WebpackCLI.loadConfig (/tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js:1460:68)
#51 3.174     at WebpackCLI.createCompiler (/tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js:1781:33) {
#51 3.174   code: 'MODULE_NOT_FOUND',
#51 3.174   requireStack: [
#51 3.174     '/tmp/grafana/node_modules/webpack-cli/lib/webpack-cli.js',
#51 3.174     '/tmp/grafana/node_modules/webpack-cli/lib/bootstrap.js',
#51 3.174     '/tmp/grafana/node_modules/webpack-cli/bin/cli.js',
#51 3.174     '/tmp/grafana/node_modules/webpack/bin/webpack.js'
#51 3.174   ]
#51 3.174 }
```

### What did you expect to happen?

Success

### Did this work before?

yes

### How do we reproduce it?

```sh
docker buildx build - \
  --platform linux/amd64 \
  --build-arg BINGO=false \
  --build-arg WIRE_TAGS=oss \
  --build-arg GO_IMAGE="golang:1.21.6-bullseye" \
  --build-arg COMMIT_SHA="$(git rev-parse HEAD)" \
  --build-arg BUILD_BRANCH="$(git rev-parse --abbrev-ref HEAD)" \
  --tag grafana/grafana:dev
```

### Is the bug inside a dashboard panel?

no

### Environment (with versions)?

Grafana: f8275b52aa4e64cd1b378d1e0da9a8784526517f
OS:
Browser:


### Grafana platform?

None

### Datasource(s)?

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,11 +14,12 @@ ENV NODE_OPTIONS=--max_old_space_size=8000
 
 WORKDIR /tmp/grafana
 
-COPY package.json yarn.lock .yarnrc.yml ./
+COPY package.json project.json nx.json yarn.lock .yarnrc.yml ./
 COPY .yarn .yarn
 COPY packages packages
 COPY plugins-bundled plugins-bundled
 COPY public public
+COPY LICENSE ./
 
 RUN apk add --no-cache make build-base python3
 
@@ -77,7 +78,6 @@ COPY pkg pkg
 COPY scripts scripts
 COPY conf conf
 COPY .github .github
-COPY LICENSE ./
 
 ENV COMMIT_SHA=${COMMIT_SHA}
 ENV BUILD_BRANCH=${BUILD_BRANCH}

--- a/Dockerfile
+++ b/Dockerfile
@@ -26,7 +26,6 @@ RUN apk add --no-cache make build-base python3
 RUN yarn install --immutable
 
 COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js ./
-COPY public public
 COPY scripts scripts
 COPY emails emails
 
@@ -179,7 +178,7 @@ RUN if [ ! $(getent group "$GF_GID") ]; then \
 
 COPY --from=go-src /tmp/grafana/bin/grafana* /tmp/grafana/bin/*/grafana* ./bin/
 COPY --from=js-src /tmp/grafana/public ./public
-COPY --from=go-src /tmp/grafana/LICENSE ./
+COPY --from=js-src /tmp/grafana/LICENSE ./
 
 EXPOSE 3000
 

